rules/aws/persistence_route_53_domain_transferred_to_another_account.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/05/10"
+maturity = "development"
+updated_date = "2021/05/10"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = "Identifies when a request has been made to transfer a Route 53 domain to another AWS account."
+from = "now-10m"
```

Comment:
we could add false positive info after the description as well - 
```
false_positives = [
    """
    A domain may be transferred to another AWS account by a system or network administrator. Verify whether the user
    identity, user agent, and/or hostname should be making changes in your environment. Domain transfers from unfamiliar
    users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the
    rule.
    """,
]
````

Reply:
:white_check_mark: